frontend/src/pages/Setup/IntegrationBar.tsx

```diff
@@ -66,15 +66,18 @@ export const IntegrationBar: React.FC<Props> = ({ integrationData }) => {
 	const integrated = integrationData?.integrated
 	const ctaText = CTA_TITLE_MAP[area!]
 
+	const start = useMemo(() => moment().subtract(30, 'days').toISOString(), [])
+	const end = useMemo(() => moment().toISOString(), [])
```

Comment:
Future improvement - So currently this is only calculated once? wondering if we should add some type of polling in case the user is sitting on the page waiting?

Reply:
@SpennyNDaJets I ended up addressing this in [6e6e08d](https://github.com/highlight/highlight/pull/7912/commits/6e6e08d257e71c634a60a687c87b932f9a26bcec)!